### PR TITLE
Add cpu_count, cpu_percent, nano_cpus parameters to container HostConfig.

### PR DESCRIPTION
**Pull request by AlexeyRokhin**
Hi.
I've recreated request because of my troubles with merging
Added ability to set new cpu parameters (cpu_count, cpu_percent, cpus) for container.

Signed-off-by: Alexey Rokhin <arokhin@mail.ru>

### CODE REVIEW

File: docker/models/containers.py [hunk 1]
```diff
@@ -456,10 +456,13 @@ def run(self, image, command=None, stdout=True, stderr=False,
             cap_add (list of str): Add kernel capabilities. For example,
                 ``["SYS_ADMIN", "MKNOD"]``.
             cap_drop (list of str): Drop kernel capabilities.
+            cpu_count (int): CPU count (Windows only).
+            cpu_percent (int): CPU percent (Windows only).
```
Comment (shin-) [inline docker/models/containers.py]: Please replace with:

> Usable percentage of the available CPUs (Windows only).

File: docker/models/containers.py [hunk 2]
```diff
@@ -456,10 +456,13 @@ def run(self, image, command=None, stdout=True, stderr=False,
...
             cpu_period (int): The length of a CPU period in microseconds.
             cpu_quota (int): Microseconds of CPU time that the container can
                 get in a CPU period.
             cpu_shares (int): CPU shares (relative weight).
+            cpus (float): Number of CPUs.
```
Comment (shin-) [inline docker/models/containers.py]: Let's conform with the API here and use

> `nano_cpus` (int): CPU quota in units of 10-9 CPUs.

File: docker/types/containers.py [hunk 3]
```diff
@@ -433,6 +434,30 @@ def __init__(self, version, binds=None, port_bindings=None,
...
+
+            self['CpuPercent'] = cpu_percent
+
+        if cpus:
+            if not isinstance(cpus, (float, int)):
+                raise host_config_type_error('cpus', cpus, 'float')
+            if version_lt(version, '1.25'):
+                raise host_config_version_error('cpus', '1.25')
+
+            self['NanoCpus'] = int(1000000000 * cpus)
```
Comment (shin-) [inline docker/types/containers.py]: Let's have `nano_cpus` be the parameter instead, limit the acceptable types to `int` and avoid `float` uncertainty altogether.

**Comment by AlexeyRokhin** [comment]
@shin- 
Done. Please check.
Should I do anything with your review?

**Comment by AlexeyRokhin** [comment]
Preparation for https://github.com/docker/compose/issues/4582

**Pull request merged**